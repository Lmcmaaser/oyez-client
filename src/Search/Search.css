/*mobile view*/
h3 {
  display: flex;
  justify-content: center;
  margin: 0px 0px 10px 0px;
  font-size: 28px;
}

.Search {
  display: flex;
  flex-direction: column;
  margin: 10px;
  font-size: 22px;
}

.instructions {
  display: flex;
  flex-direction: column;
  margin: 10px;
}
.info {
  display: flex;
  flex-direction: column;
  /* align-items: center; */
  justify-content: center;
  padding: 5px;
  margin: 0px;
  background-color: #B0AFE3;
  border: 5px solid #B0AFE3;
  border-radius: 8px;
}

/*desktop view*/
@media screen and (min-width: 600px) {
  h3 {
    justify-content: flex-start;
    margin: 10px 0px 10px 0px;
  }

  .instructions {
    display: flex;
    flex-direction: column;
    margin: 10px;
  }

  .info {
    display: flex;
    flex-direction: column;
    /* align-items: center; */
    justify-content: center;
    padding: 5px;
    margin: 0px;
    background-color: #B0AFE3;
    border: 5px solid #B0AFE3;
    border-radius: 8px;
  }
}
