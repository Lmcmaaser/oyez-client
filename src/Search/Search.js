import React from 'react';
import { Link } from 'react-router-dom';
import Content from '../content';
import { GrBarChart } from "react-icons/gr";
import { GrLineChart } from "react-icons/gr";
import { GrPieChart } from "react-icons/gr";

export default class Search extends React.Component {

  render () {

    return (
      <div>
        <div className="instructions">
          <Content className='Search'>
            <h3>Get Data!</h3>
            <p className="info">Select what data you would like to see:</p>
              <p className="info">View all reports submitted to Oyez by state:
                <Link to='/all'>
                  <GrBarChart />
                </Link>
              </p>
              <p className="info">View new reports over time by state:
                <Link to='/state'>
                  <GrLineChart />
                </Link>
              </p>
              <p className="info">View by zipcode:
                <Link to='/zipcode'>
                  <GrPieChart />
                </Link>
              </p>
          </Content>
        </div>
      </div>
    )
  }
}
