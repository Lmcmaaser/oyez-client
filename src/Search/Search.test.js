import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Search from './Search';

it('renders a .instructions by default', () => {
  const wrapper = shallow(<Search />);
  expect(toJson(wrapper)).toMatchSnapshot();
});

it('renders without crashing', () => {
  shallow(<Search />);
});
