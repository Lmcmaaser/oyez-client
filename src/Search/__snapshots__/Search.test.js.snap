// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a .instructions by default 1`] = `
<div>
  <div
    className="instructions"
  >
    <Content
      className="Search"
    >
      <h3>
        Get Data!
      </h3>
      <p
        className="info"
      >
        Select what data you would like to see:
      </p>
      <p
        className="info"
      >
        View all reports submitted to Oyez by state:
        <Link
          to="/all"
        >
          <GrBarChart />
        </Link>
      </p>
      <p
        className="info"
      >
        View new reports over time by state:
        <Link
          to="/state"
        >
          <GrLineChart />
        </Link>
      </p>
      <p
        className="info"
      >
        View by zipcode:
        <Link
          to="/zipcode"
        >
          <GrPieChart />
        </Link>
      </p>
    </Content>
  </div>
</div>
`;
